internal/impl/io/input_websocket.go

```diff
@@ -154,14 +163,17 @@ func (w *websocketReader) Connect(ctx context.Context) error {
 	}
 
 	var client *websocket.Conn
+
+	dialer := *websocket.DefaultDialer
```

Comment:
This is an interesting gotcha. When `w.tlsEnabled == true`, the existing implementation created a new `websocket.Dialer` instance instead of copying `websocket.DefaultDialer`, which [sets](https://github.com/gorilla/websocket/blob/01b0aaed92c975e5e740c6202e7ff8e5a85085ce/client.go#L144-L147) `Proxy: http.ProxyFromEnvironment` and `HandshakeTimeout: 45 * time.Second`. I think a shallow copy should suffice in this case.